com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -861,14 +861,18 @@ static int GetMainLightIndex(UniversalRenderPipelineAsset settings, NativeArray<
                 if (currLight == null)
                     break;
 
-                if (currLight == sunLight)
-                    return i;
-
-                // In case no shadow light is present we will return the brightest directional light
-                if (currVisibleLight.lightType == LightType.Directional && currLight.intensity > brightestLightIntensity)
+                if (currVisibleLight.lightType == LightType.Directional)
                 {
-                    brightestLightIntensity = currLight.intensity;
-                    brightestDirectionalLightIndex = i;
+                    // Sun source needs be a directional light
+                    if (currLight == sunLight)
+                        return i;
+
+                    // In case no shadow light is present we will return the brightest directional light
```

Comment:
nitpicking here I think. :)
Is this comment correct? I guess it means to say no sun light?